### PR TITLE
Fixes the error "iconv: invalid option -- o" by redirecting stdout to the desired file. 

### PR DESCRIPTION
**Pull request by alyxdeburca**
Was having issues building on MacOS (presumably due to some weird Apple specific version of iconv) but changing this line allowed the build to complete successfully

### CODE REVIEW

**Comment by Tmcg2** [comment]
Looks like BSD iconv doesn't have the `-o` option at all, which would explain why It doesn't work on MacOS. Good find.

**Comment by alyxdeburca** [comment]
I'll keep checking for more weird MacOS only behaviours

File: .gitignore [hunk 1]
```diff
@@ -82,6 +82,7 @@ expected/*
 !/sound/**/*custom*/**/*.aiff
 !/assets/**/*custom*.bin
 !/assets/**/*custom*/**/*.bin
-
+tools/*.md
 # Doxygen
-docs/html/*
+tools/doxygen_syms.md
+tools/doxygen_syms.md
```
Comment (MegaMech) [inline .gitignore:88]: Please revert this change. We need doxygen_syms generated in the PRs for the github pages API site.

**Review by MegaMech — CHANGES_REQUESTED** [review]

**Comment by MegaMech** [comment]
Could someone test on Linux?

**Comment by alyxdeburca** [comment]
Spinning up a vm now :)

File: .gitignore (hunk 1 above)
Comment (alyxdeburca) [reply .gitignore]: Yeah VSCode autmatically did that. Reverting now

**Comment by alyxdeburca** [comment]
I fucked up my local branch. Feel free to fix any git related issues with the last few commits

**Comment by MegaMech** [comment]
> I fucked up my local branch. Feel free to fix any git related issues with the last few commits

Generally, creating a new branch and doing changes in there helps prevent issues with git. And then the master branch is never modified and just synced with the base repo

File: Makefile [hunk 2]
```diff
@@ -561,11 +562,14 @@ COURSE_DATA_TARGETS := $(foreach dir,$(COURSE_DIRS),$(BUILD_DIR)/$(dir)/course_d
...
 $(BUILD_DIR)/%.jp.c: %.c
 	$(call print,Encoding:,$<,$@)
-	iconv -t EUC-JP -f UTF-8 $< -o $@
-
+  ifeq ($(UNAME_S), Darwin)
+	  iconv -t EUC-JP -f UTF-8 $< > $@
+  endif
+  ifeq ($(UNAME_S), Darwin)
+    iconv -t EUC-JP -f UTF-8 $< -o $@
+  endif
```
Comment (MegaMech) [inline Makefile:572]: I think you might not work Darwin on the second one here? the -o should be non-mac and the > should be mac right?

**Review by MegaMech — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (alyxdeburca) [reply Makefile:572]: Oh yeah, did I copy and pasted and forgot to change that part to Linux instead of darwin

File: .gitignore (hunk 1 above)
Comment (alyxdeburca) [reply .gitignore]: @MegaMech can you confirm that this reverted? It's not showing right for me

**Comment by MegaMech** [comment]
I forgot that if the builder passes then it does work on Linux. My bad.

**Comment by MegaMech** [comment]
Is it possible the > works on Linux as well as mac? If it works on both...

**Comment by MegaMech** [comment]
Ya, you're original PR did pass.

**Comment by alyxdeburca** [comment]
My git is so fucked up. I have no idea if this is pushing right

**Comment by MegaMech** [comment]
You could try starting over.

Or try editing the file right in github. You should be able to fix it that way pretty easily.

Click > Files Changed. Find the makefile --> click the three dots --> click edit.

For some reason you have some git conflict merge artifacts in your PR now.

**Comment by MegaMech** [comment]
I fixed it for using in the browser using the edit button.

Lets see if the builder passes

**Review by MegaMech — APPROVED** [review]

**Comment by MegaMech** [comment]
Hopefully all the other platforms support this command...
